api/src/test/java/org/openmrs/validator/ConditionValidatorTest.java

```diff
@@ -47,23 +53,31 @@ public void setUp(){
 	public void shouldFailIfGivenNull(){
 		expectedException.expect(IllegalArgumentException.class);
 		expectedException.expectMessage(NULL_ERROR_MESSAGE);
-
 		validator.validate(null, errors);
 	}
 
 	@Test
 	public void shouldFailIfGivenInstanceOfClassOtherThanCondition(){
 		expectedException.expect(IllegalArgumentException.class);
 		expectedException.expectMessage(INCOMPATIBLE_ERROR_MESSAGE);
-
 		validator.validate(new Cohort(), errors);
 	}
+	
+	@Test
+	public void shouldFailIfGivenConditionWithNullConditionParameters(){
```

Comment:
In the method name, change parameters to properties

Reply:
Okay